internal/backend/conjur/ssh_templates.go

```diff
@@ -0,0 +1,24 @@
+package conjur
```

Comment:
So we will not be writing the SSH certificates to the backend?

Reply:
That's correct, there is no centralized system for SSH key management. There is a concept called KRL that is similar to the CRL, but it is defined on each local system for the SSH server to manage.